tests/utils/test_dict_utils.py

```diff
@@ -0,0 +1,35 @@
+from scout.utils.dict_utils import remove_empty_list, remove_nonetype
+
+
+def test_remove_nonetype_no_change():
+    # WHEN a dict *not* containing a None value
+    d = {"a": "1", "b": 2, "c": 3}
+
+    # THEN calling removeNoneValues(dict) will not change dict
+    assert d == remove_nonetype(d)
+
+
+def test_remove_nonetype():
+    # WHEN a dict containing a value which is None
+    d = {"a": "1", "b": 2, "c": None}
+
+    # THEN calling removeNoneValues(dict) will remove key-value pair
+    # where value=None
+    assert {"a": "1", "b": 2} == remove_nonetype(d)
+
+
+def test_remove_empty_list_no_change():
+    # WHEN a dict *not* containing an empty list, []
+    d = {"a": "1", "b": 2, "c": 3}
```

Comment:
```suggestion
    d = {"a": "1", "b": 2, "c": [3]}
```

Reply:
It's fine really, but perhaps it's a better test if you provide an element with an array?

Reply:
Good idea. Changes are being pushed.

Reply:
It's good to merge I think!